### ISSUE TITLE
Zombie processes grow forever on agent

### ISSUE THREAD

**Issue by mrmattyboy1** [DockStudios/jmon#49]
After a week or so, there are 5K zombie processes on the agent. Checks are using firefox.

**Comment by mrmattyboy1** [comment]
Does selenium clear up the browser and geckodriver without the Pid termination (https://github.com/DockStudios/jmon/blob/main/jmon/runner.py#L100)?

Investigate: https://psutil.readthedocs.io/en/latest/#kill-process-tree

**Comment by mrmattyboy1** [comment]
Example:
```
> ps aux | grep defu
root     13665  0.0  0.0      0     0 ?        Z    08:04   0:00 [Socket Process] <defunct>
root     13715  0.0  0.0      0     0 ?        Z    08:04   0:00 [dbus-launch] <defunct>
root     13717  0.0  0.0      0     0 ?        Zs   08:04   0:00 [dbus-daemon] <defunct>
root     13769  0.0  0.0      0     0 ?        Z    08:04   0:00 [Utility Process] <defunct>
```

**Comment by mrmattyboy1** [comment]
Possibly see: https://stackoverflow.com/questions/38518998/selenium-leaves-behind-running-processes

**Comment by MatthewJohn** [comment]
Fixed in release 4.14.1
